### ISSUE TITLE
Flash frame/taskbar icon on terminal bell

### ISSUE THREAD

**Issue by sWski** [Eugeny/tabby#8981]
**Is your feature request related to a problem? Please describe.**
I am missing this feature which other terminals (Windows Terminal, putty) have - when bell comes and the terminal application does not have keyboard focus (it is minimized or another application is selected), it flashes the taskbar icon (on Windows).

**Describe the solution you'd like**
When bell comes, Tabby should check config whether to flash frame and call relevant function. In Electron this can be achieved by [this api](https://www.electronjs.org/docs/latest/tutorial/windows-taskbar#flash-frame). It seems it's working on macOS as well (bouncing icon), don't know about Linux.

In addition, it would be good to change the Tabby settings to allow the user to choose more responses to the bell. See attached screenshot from Windows Terminal bellow.

**Describe alternatives you've considered**
There is no alternative but to not use Tabby 😄 

**Additional context**
![image](https://github.com/Eugeny/tabby/assets/10461762/b0032b16-afab-4547-8519-3dc4439c3efc)
Windows Terminal bell settings - *Flash window* is *Visual* in Tabby, *Flash taskbar* is this feature request

**Comment by nscavalier** [comment]
this is a great feature for claude code like agents.

**Comment by jmichler** [comment]
It would be great to have this. I often start long-running operations and would then like to know once that is done. For this I have a wrapper script around my command that at the end triggers a bell. This works great in the builtin mac os terminal; but no way to get the notification flashing in the dock or in the notification center for tabby. One of the very few features I'm missing
